### ISSUE TITLE
Table border not showing while saving content as html

### ISSUE THREAD

**Issue by vikaspathneja** [froala/wysiwyg-editor#3686]
##### Expected behavior.
While using table in Froala Editor and tried to save its content as html , table border should come along with the content

##### Actual behavior.
table border is not showing ,only table content is showing ,

##### Screenshots
 **added table**
![first](https://user-images.githubusercontent.com/20497643/65938473-a5127180-e440-11e9-9938-86173079fa0e.PNG)

**inline border not added**
![second](https://user-images.githubusercontent.com/20497643/65938198-fa01b800-e43f-11e9-99b7-874bb3b41484.PNG)

**impact of inline border not added ,border not showing** 
![third](https://user-images.githubusercontent.com/20497643/65938199-fa01b800-e43f-11e9-98cc-b97f0b6c71aa.PNG)

**when added border="1" in html**
![fourth](https://user-images.githubusercontent.com/20497643/65938733-747f0780-e441-11e9-9a67-2bc7beda5848.PNG)

**now border is showing in html** 
![fifth](https://user-images.githubusercontent.com/20497643/65938664-2ff36c00-e441-11e9-90f7-08142fab54e3.PNG)

###Suggested 
Like fonts and other styles are inline within html when used , same way when applied dashed-border or alternate table style it should also inline so that when saving as html it should not depend on css/js dependencies

Please look into this as this is very basic feature and requirement in projects

**Comment by harshit-budhraja** [comment]
@vikaspathneja Was this issue resolved? Did anyone get in touch with you about this?
Seems like I'm facing the exact same issue.

**Comment by vikaspathneja** [comment]
No, not yet, issue is still open and i think no one is looking into so i switched to another editor

**Comment by Kiraboi** [comment]
I am also facing the same issue.

**Comment by dejanmartinovic** [comment]
Table element does not have a visible border by default: https://www.w3schools.com/html/tryit.asp?filename=tryhtml_table.
If you want to have visible borders outside the editor you can follow one of the following approaches:
1. Wrap the HTML content with `fr-view` class (as described in the Get Started guide: https://froala.com/wysiwyg-editor/docs/overview/#display)
2. Create a custom button, apply the inline style, and place it in the Table Edit toolbar: https://jsfiddle.net/dejan_m/og0hxf2t/4257/

**Comment by dqfan2012** [comment]
Hey guys - would there be a way in the Froala options to allow one of the tableStyles options classes to be selected by default (so that my end users don't have to manually select it each time)?

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
